marketplace/mux-video-uploader/src/index.tsx

```diff
@@ -101,6 +113,53 @@ export class App extends React.Component<AppProps, AppState> {
     return headers;
   };
 
+  requestDeleteAsset = async () => {
+    if (!this.state.value || !this.state.value.assetId) {
+      throw Error(
+        'Something went wrong, we cannot delete an asset without an assetId.'
+      );
+    }
+
+    const result = await this.props.sdk.dialogs.openConfirm({
+      title: "Are you sure you want to delete this asset?",
+      message: "This will remove the asset in Mux and in Contentful. There is no way to recover your video, make sure you have a backup if you think you may want to use it again.",
+      intent: "negative",
+      confirmLabel: "Yes, delete this asset",
+      cancelLabel: "Nevermind",
+    });
+
+    if (!result) {
+      this.setState({ isDeleting: false });
+      return;
+    }
+    this.setState({ isDeleting: true });
+
+    const res = await fetch(`https://api.mux.com/video/v1/assets/${this.state.value.assetId}`, {
+      ...this.muxBaseReqOptions,
+      method: 'DELETE',
+    });
+
+    if (res.status === 401) {
+      throw Error(
+        'Looks like something is wrong with the Mux Access Token specified in the config. Are you sure the token ID and secret in the extension settings match the access token you created?'
+      );
```

Comment:
I noticed that you `throw` in this method and elsewhere. Are these message you want to show to the user? It might make more sense to use `sdk.notifier.error` which shows a red error banner in the UI instead of in the dev console.

More information about it can be found here: 
https://www.contentful.com/developers/docs/extensibility/ui-extensions/sdk-reference/#notifier

Reply:
Yes, the intention is meant for this error to be shown! We found that by throwing errors the extension does in fact render them with the red error banner in the UI (see screenshot below).

Is `sdk.notifier.error` the preferred method?

![looks-like_2020-01-27_14-48-47](https://user-images.githubusercontent.com/764988/73220829-30142780-4114-11ea-9333-885d72752990.png)

Reply:
Yes, the intention is meant for this error to be shown! We found that by throwing errors the extension does in fact render them with the red error banner in the UI (see screenshot below).

![looks-like_2020-01-27_14-48-47](https://user-images.githubusercontent.com/764988/73220829-30142780-4114-11ea-9333-885d72752990.png)

Is `sdk.notifier.error` the preferred method?

Reply:
Ah, I didn't catch that these errors would go into this banner. We usually use `sdk.notifier.error` for showing errors but I think that these inline errors (like in your screenshot) are even better because they narrow down where the error occurred.